package.json

```diff
@@ -66,8 +66,7 @@
   "devDependencies": {
     "@babel/core": "^7.10.5",
     "@babel/preset-react": "^7.10.4",
-    "@storybook/addon-docs": "^6.1.5",
-    "@storybook/addon-viewport": "^6.1.5",
```

Comment:
These two dependencies are mentioned in [`.github/dependabot.yml`](https://github.com/trussworks/react-uswds/blob/main/.github/dependabot.yml). Can we update it to reference '@storybook/addon-essentials"?

Reply:
done